### ISSUE TITLE
List1 XXX

### ISSUE THREAD

**Issue by kohler** [nathanielherman/sto#15]
Can we delete List1? I mean obviously we can. :) So: What are the reasons to NOT delete List1?

**Comment by jinala** [comment]
I think List1 is the version that also tracks updates to the list items.
All of its data items are enclosed in a TBox.

**Comment by huangyihe** [comment]
Yeah, `List1` is a "real" list :) `List.hh` is the STAMP list set I think.

**Comment by kohler** [comment]
It's just that List1 wasn't used anywhere.

**Comment by huangyihe** [comment]
No we're not using it, except in `list1.cc`. My snapshot list implementation follows `List1.hh` though.
